Sources/NIOPosix/BSDSocketAPIPosix.swift

```diff
@@ -212,10 +212,10 @@ extension NIOBSDSocket {
 
     // The protocol subtype for MPTCP.
     // Returns nil if mptcp is not supported.
-    static var mptcpProtocolSubtype: Int? {
+    static var mptcpProtocolSubtype: NIOBSDSocket.ProtocolSubtype? {
```

Comment:
Should we remove this in favour of `NIOBSDSocket.ProtocolSubtype.mptcp`?

Reply:
Done in fadd5c120b7626be22b7731daab94c171a1fe610